cpmpy/expressions/globalfunctions.py

```diff
@@ -229,30 +232,51 @@ class Element(GlobalFunction):
         Solvers implement it as Arr[Idx] == Y, but CPMpy will automatically derive or create
         an appropriate Y. Hence, you can write expressions like Arr[Idx] + 3 <= Y
 
+        CPMpy supports multi-dimensional Element constaints such as arr[a,b] == 10
+
         Element is a CPMpy built-in global constraint, so the class implements a few more
         extra things for convenience (.value() and .__repr__()). It is also an example of
         a 'numeric' global constraint.
     """
 
-    def __init__(self, arr, idx):
-        if is_boolexpr(idx):
-            raise TypeError("index cannot be a boolean expression: {}".format(idx))
-        if is_any_list(idx):
-            raise TypeError("For using multiple dimensions in the Element constraint, use comma-separated indices")
-        super().__init__("element", [arr, idx])
+    def __init__(self, arr, *idx):
```

Comment:
I'm not sure I like idx being a *idx implicit array...

my impression was that numpy is more natural in it, that an index can also be a tuple instead of a single value; and in case of a tuple, the index is the tuple, not that there are multiple indices.

So then here, and in all below, it would be (self, arr, idx) and still super().ini("el", [arr, idx]) etc...?